frontend/src/components/common/GeneralAccordionButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { AccordionButton, AccordionIcon, Box, Text } from "@chakra-ui/react";
+
+type GeneralAccordionButtonProps = {
+  title: string;
+};
+
+const GeneralAccordionButton = ({
+  title,
+}: GeneralAccordionButtonProps): React.ReactElement => {
+  return (
+    <AccordionButton
+      borderRadius="20px"
+      bg="primary.green.600"
+      _hover={{ bg: "primary.green.600" }}
```

Comment:
how come the hover and normal bg colors are the same? shouldn't the hover be a shade darker?

Reply:
I think that was just in the initial design so I left it the same. The `GeneralAccordionButton` component itself is just an abstraction of a component we used before, so I didn't want to make any changes at all.